python/llm/example/GPU/Pipeline-Parallel-Inference/README.md

```diff
@@ -0,0 +1,83 @@
+# Run BigDL-LLM on Multiple Intel GPUs in pipeline parallel fashion
+
+This example demonstrates how to run BigDL-LLM optimized low-bit model vertically partitioned on two [Intel GPUs](../README.md).
+
+## Requirements
+To run this example with BigDL-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information. For this particular example, you will need at least two GPUs on your machine.
+
+## Example:
+
+### 1.1 Install BigDL-LLM
+
+```bash
+conda create -n llm python=3.9
+conda activate llm
+# below command will install intel_extension_for_pytorch==2.1.10+xpu as default
+# you can install specific ipex/torch version for your need
+pip install --pre --upgrade bigdl-llm[xpu_2.1] -f https://developer.intel.com/ipex-whl-stable-xpu
+# configures OneAPI environment variables
+source /opt/intel/oneapi/setvars.sh
+
+conda install -c conda-forge -y gperftools=2.10 # to enable tcmalloc
+```
+
+### 1.2 Build and install patched version of Intel Extension for PyTorch (IPEX)
+
+```bash
+conda activate llm
+source /opt/intel/oneapi/setvars.sh
+git clone https://github.com/yangw1234/intel-extension-for-pytorch.git
```

Comment:
https://github.com/intel/intel-extension-for-pytorch/tree/xpu-main may also work.